pkg/render/logstorage.go

```diff
@@ -450,20 +451,6 @@ func (es elasticsearchComponent) elasticsearchExternalService() *corev1.Service
 	}
 }
 
-func (es elasticsearchComponent) kibanaExternalService() *corev1.Service {
```

Comment:
This should have been removed earlier as well since it is no longer used given that Managed clusters don't have any components configured to hit this service.